src/map/modifier.h

```diff
@@ -356,6 +357,7 @@ enum class Mod
     DESPOIL           = 896, // Increases THF Despoil chance
     PERFECT_DODGE     = 883, // Increases Perfect Dodge duration in seconds
     TRIPLE_ATTACK     = 302, // Percent chance
+    TRIPLE_ATTACK_DMG = 287, // Increases "Triple Attack" damage/"Triple Attack" damage + (in percents, e.g. +20 = +20% damage)
```

Comment:
287 is already used by `ATMA`

See the bottom of the enum:

`// SPARE = 1038, and onward`

Suggest using 1039, and updating SPARE to 1040